src/pages/home/sidebar/SidebarScreen/FloatingActionButtonAndPopover.js

```diff
@@ -81,7 +83,12 @@ class FloatingActionButtonAndPopover extends React.Component {
     }
 
     componentDidMount() {
-        const routes = lodashGet(this.props.navigation.getState(), 'routes', []);
+        const navigationState = this.props.navigation.getState();
+        const currentRoute = navigationState.routes[navigationState.index];
+        if(currentRoute && ![NAVIGATORS.CENTRAL_PANE_NAVIGATOR, SCREENS.HOME].includes(currentRoute.name)) {
```

Comment:
Missing space after `if`.

```suggestion
        if (currentRoute && ![NAVIGATORS.CENTRAL_PANE_NAVIGATOR, SCREENS.HOME].includes(currentRoute.name)) {
```